core/templates/pages/collection-player-page/collection-player-page.directive.html

```diff
@@ -1,6 +1,11 @@
 <div>
   <background-banner></background-banner>
-  <div ng-if="$ctrl.collection" class="oppia-collection-player-tiles-container">
+  <div ng-if="!$ctrl.collection.getCollectionNodeCount()" class="no-exploration-created">
+    <div class="oppia-collection-path-section">
+      No exploration has been created yet.
```

Comment:
Hm, let's change this to "No Exploration has been added to this Collection".

Otherwise it seems a bit misleading.

Reply:
Done.